octoprint_enclosure/__init__.py

```diff
@@ -978,45 +978,58 @@ def update_ui_inputs(self):
     def get_sensor_data(self, sensor):
         try:
             if self.development_mode:
-                temp, hum = self.read_dummy_temp()
+                temp, hum, airquality = self.read_dummy_temp()
             else:
                 if sensor['temp_sensor_type'] in ["11", "22", "2302"]:
                     temp, hum = self.read_dht_temp(sensor['temp_sensor_type'], sensor['gpio_pin'])
                 elif sensor['temp_sensor_type'] == "18b20":
                     temp = self.read_18b20_temp(sensor['ds18b20_serial'])
                     hum = 0
+					airquality = 0

```

Comment:
the indentation for `airquality` is incorrect in this file (it has an extra space)

Reply:
Thanks for the review. I fixed both indent errors, hope it's all correct (I'm not all too familiar with github + haven't even run the code yet)

Reply:
no, i still see incorrect indentation

Reply:
Darn, those tabs :/